superset/translations/pt/LC_MESSAGES/messages.json

```diff
@@ -8,6 +8,229 @@
         "plural_forms": "nplurals=2; plural=(n != 1)",
         "lang": "pt"
       },
+<<<<<<< HEAD
+      "Home": [
+        ""
+      ],
+      "Annotation Layers": [
+        "Camadas de anotação"
+      ],
+      "Manage": [
+        "Gerir"
+      ],
+      "Databases": [
+        "Bases de dados"
+      ],
+      "Data": [
+        "Base de dados"
+      ],
+      "Datasets": [
+        "Bases de dados"
```

Comment:
don't think these should all be the same, "conjuntos de dados" ? ou deixar Datasets?